tests/storage/hotplug.go

```diff
@@ -863,9 +864,48 @@ var _ = SIGDescribe("Hotplug", func() {
 					},
 				}, false, ""))
 				Expect(err).To(HaveOccurred())
-				Expect(err.Error()).To(ContainSubstring("conflicts with an existing volume of the same name on the vmi template"))
+				Expect(err.Error()).To(ContainSubstring("Unable to add volume [disk0] because volume with that name already exists"))
 			})
 
+			It("should reject hotplugging the same volume with an existing volume name", func() {
+				dvBlock := createDataVolumeAndWaitForImport(sc, corev1.PersistentVolumeBlock)
```

Comment:
I don't think this will work in a WFFC scenario. Since the code wait for the DV to be ready which will never happen. Maybe just create it and assume it is ready (blank ones usually are before you attempt to hotplug them)

Reply:
@awels the create adds to the dv `force bind` annotation so it should be fine right?